app/concepts/reservations/use_cases/create.rb

```diff
@@ -0,0 +1,15 @@
+module Reservations
+  module UseCases
+    class Create
+      attr_reader :repository, :params
```

Comment:
I don't think, that you need to declare `:params` here.

Reply:
+1, you don't use it here in this class